config/checkstyle/checkstyle.xml

```diff
@@ -313,5 +313,7 @@
         default="checkstyle-xpath-suppressions.xml"/>
       <property name="optional" value="true"/>
     </module>
+    <module name="SuppressWarningsHolder"/>
```

Comment:
Allowing us to suppress checkstyle warnings for things that we want to keep as they are